packages/presets/src/ai/messages/text.ts

```diff
@@ -131,6 +131,12 @@ export class AIAnswerText extends WithDisposable(LitElement) {
       rich-text .nowrap-lines v-element span {
         white-space: pre;
       }
+      editor-host:focus-visible {
+        outline: none;
+      }
+      editor-host * {
+        box-sizing: border-box;
```

Comment:
Similar to
https://github.com/toeverything/blocksuite/blob/423c98e6667e6cbfe92a8cf8c16a5761f07d8e5e/packages/presets/src/editors/editor-container.ts#L120-L122